### ISSUE TITLE
function 'XGDMatrixSetStrFeatureInfo' not found

### ISSUE THREAD

**Issue by bigjyh1979** [dmlc/xgboost#5900]
Hello,
I installed the xgboost-gpu version, but when I run the test code as below, got the error "function 'XGDMatrixSetStrFeatureInfo' not found".
Does anyone know how to figure it out? Thanks~

I tried python 3.6 and 3.7 but got the same result.
CUDA : 10.0

```
import xgboost as xgb
from sklearn.datasets import load_boston

boston = load_boston()

# XGBoost API example
params = {'tree_method': 'gpu_hist', 'max_depth': 3, 'learning_rate': 0.1}
dtrain = xgb.DMatrix(boston.data, boston.target)
xgb.train(params, dtrain, evals=[(dtrain, "train")])

# sklearn API example
gbm = xgb.XGBRegressor(silent=False, n_estimators=10, tree_method='gpu_hist')
gbm.fit(boston.data, boston.target, eval_set=[(boston.data, boston.target)])
```

output
```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-1-94717a6e6cf2> in <module>
      6 # XGBoost API example
      7 params = {'tree_method': 'gpu_hist', 'max_depth': 3, 'learning_rate': 0.1}
----> 8 dtrain = xgb.DMatrix(boston.data, boston.target)
      9 xgb.train(params, dtrain, evals=[(dtrain, "train")])
     10 

~\anaconda3\envs\finlabgpu\lib\site-packages\xgboost\core.py in __init__(self, data, label, weight, base_margin, missing, silent, feature_names, feature_types, nthread)
    442         self.set_info(label=label, weight=weight, base_margin=base_margin)
    443 
--> 444         self.feature_names = feature_names
    445         self.feature_types = feature_types
    446 

~\anaconda3\envs\finlabgpu\lib\site-packages\xgboost\core.py in feature_names(self, feature_names)
    770         else:
    771             # reset feature_types also
--> 772             _check_call(_LIB.XGDMatrixSetStrFeatureInfo(
    773                 self.handle,
    774                 c_str('feature_name'),

~\anaconda3\envs\finlabgpu\lib\ctypes\__init__.py in __getattr__(self, name)
    359         if name.startswith('__') and name.endswith('__'):
    360             raise AttributeError(name)
--> 361         func = self.__getitem__(name)
    362         setattr(self, name, func)
    363         return func

~\anaconda3\envs\finlabgpu\lib\ctypes\__init__.py in __getitem__(self, name_or_ordinal)
    364 
    365     def __getitem__(self, name_or_ordinal):
--> 366         func = self._FuncPtr((name_or_ordinal, self))
    367         if not isinstance(name_or_ordinal, int):
    368             func.__name__ = name_or_ordinal

AttributeError: function 'XGDMatrixSetStrFeatureInfo' not found

```

**Comment by trivialfis** [comment]
It seems your xgboost Python package is on latest master branch, but the shared library is older.  How did you install XGBoost?

**Comment by bigjyh1979** [comment]
> It seems your xgboost Python package is on latest master branch, but the shared library is older. How did you install XGBoost?
Ah, thanks for your hint.
I installed by the pre-compiled xgboost.dll file, like here: [https://medium.com/@SeoJaeDuk/how-to-install-xgboost-gpu-no-gpu-on-window-10-x64-for-python-3-8c2ffa00fb29](url)
It doesn't work, so I tried to compile by myself through Cmake, it still has some problem.
Finally I installed by this way: [https://github.com/dmlc/xgboost/issues/5706](url)
It works for me.

Thanks~
